c10/core/Backend.h

```diff
@@ -263,13 +277,15 @@ static inline Backend backendToHammerBlade(Backend b) {
   switch (b) {
     case Backend::CPU:
       return Backend::HammerBlade;
+    case Backend::SparseCPU:
+      return Backend::SparseHammerBlade;
     case Backend::CUDA:
     case Backend::HIP:
     case Backend::HammerBlade:
     case Backend::MSNPU:
     case Backend::XLA:
-    case Backend::SparseCPU:
     case Backend::SparseCUDA:
+    case Backend::SparseHammerBlade:
```

Comment:
Hmmm. is this right? it seems we should have SparsehammerBlade to HammerBlade?

Reply:
In this way, we need to implement to_dense and to_sparse on HB then, which currently not support. I am not sure whether we need support direct transforming between HB tensor and HB sparse tensor ?

Reply:
So I need to let HammerBlade->HammerBlade and also SparseHammerBlade->Hammerblade right ?

Reply:
I'm not sure either. Maybe we could have a discussion during the meeting today?

Reply:
Sounds good !